lib/beaker/version.rb

```diff
@@ -1,5 +1,5 @@
 module Beaker
   module Version
-    STRING = '3.26.0'
+    STRING = '3.27.0'
```

Comment:
please remove version string changes from this PR. Those are handled in our releases